### PR TITLE
Fixing bugs seen in production

### PR DESCRIPTION
**Pull request by trinitroglycerin**
This pull request fixes a litany of bugs we've identified in production, old and new, as well as adding some quality of life improvements.

The commit messages should be self-describing

### CODE REVIEW

File: cli/consts.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ var Dev bool = false
 const DefaultTTL uint = 1
 
 // DefaultTimeRemaining for new key requests in minutes
-const DefaultTimeRemaining uint = 60
+const DefaultTimeRemaining uint = 5
```
Comment (trinitroglycerin) [inline cli/consts.go:20]: This was dropped to 5 minutes because the default TTL is 1 hour, which means that for any user that doesn't specify `--time-remaining` or `--ttl`, the behaviour of `keyconjurer get` will be to *always* fetch new credentials even if there are perfectly legitimate ones in the environment

**Review by JasonHusek — APPROVED** [review]
Review discussion is over at https://github.com/trinitroglycerin/key-conjurer/pull/2 
TLDR: Looks good
